src/modules/internet/util.ts

```diff
@@ -0,0 +1,19 @@
+/**
+ *
+ * @param input String to encode to Base64.
+ *
+ * @returns Base64 URL encoded string.
+ *
+ * @example const encodedHeader = toBase64Url(JSON.stringify(header));
+ */
+export function toBase64Url(input: string): string {
+  const utf8Bytes = new TextEncoder().encode(input);
+  const binaryString = Array.from(utf8Bytes, (byte) =>
+    String.fromCodePoint(byte)
+  ).join('');
```

Comment:
A little nitpicky, but if you have time this line can be written without creating a new function scope.

```suggestion
  const binaryString = Array.from(utf8Bytes, String.fromCodePoint).join('');
```

Reply:
![Screenshot 2024-06-19 at 6 17 53 PM](https://github.com/faker-js/faker/assets/19218300/25d65ad6-b231-4440-a77d-b24c561485a7)

I think it messes with the encoding. Since `String.fromCodePoint` accepts variable args, it passes both the value and the index from the `mapFn` in `Array.from`.

Reply:
I see. Thank you for trying. I learned something new today :)